tools/clean-doc-html

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+# Run this command with a list of html file names as arguments
+# When run, it will modify the files in place to remove transient parts of the documentation
+# that does not need to show up in the documentation diff
+sed -i -e '/This is documentation/ s/ built with GitHub PR .* for development/ for development/' \
+    -e 's;?v=[0-9a-f]*";";' \
+    -e '\;<script type="application/vnd\.jupyter\.widget-state+json">;,\;</script>; d' \
+    -e 's;#L[0-9]*";";' \
+    -e 's;tab-set--[0-9]*-;tab-set-;' \
+    -e 's;"tab-set--[0-9]*";"tab-set";' \
+    "$@"
```

Comment:
Not sure if this is more suitable to be put in `.ci` instead. On the other hand it seems like something that is not _that_ specific to CI?

Reply:
I suggest to put it in `.github/workflows`. `tools` is for developers (I have a hard time seeing why someone would like to run this script locally) and `.ci` is mostly outdated.

Reply:
Oops, I misreads.

On the other hand 

> .ci is mostly outdated.

the problem is the substitute aren't good enough yet. For example `test-new` takes only ≈ 15 minutes while meson test takes an hour. Meson doesn't have incremental build or documentation build or documentation diff either.

Reply:
> I suggest to put it in `.github/workflows`.

Actually… this feels wrong somehow.

Argument to support it: there are already two scripts there.

Arguments against it: we're only using GitHub actions for CI, so GitHub workflows and CI should be synonymous. It is not a good idea to have two locations for the same purpose. (if we do this we might consider moving the scripts back as well, but I guess that's out of scope here)

Reply:
In https://github.com/sagemath/sage/pull/39642, I propose to remove the `.ci` directory. What do you think?